game/scripts/vscripts/Sections/Chapter4/SectionOpening.ts

```diff
@@ -246,7 +256,8 @@ function checkIfScanCoversTheLocation(targetScanLocation: Vector, context: Tutor
             return true;
         }
         displayDotaErrorMessage(LocalizationKey.Error_Opening_1);
-        dg.playAudio(LocalizationKey.Script_4_Opening_15, LocalizationKey.Script_4_Opening_15, context[CustomNpcKeys.SlacksMudGolem]);
+        if (targetScanLocation === secondScanLocation)
+            dg.playAudio(LocalizationKey.Script_4_Opening_15, LocalizationKey.Script_4_Opening_15, context[CustomNpcKeys.SlacksMudGolem]);
```

Comment:
Is this line properly stopped when skipping? You might want to get the token (returned by this) and use `dg.stop(token)`. Didn't test myself.

Also, the dialog explaining the second scan can be missed/stopped by trying to scan prematurely in a different location. The scan should behave like the first one: ignores scan orders until it's time to scan that location.

Reply:
Regarding the interrupt of the second dialog, I guess I won't fork that dialog then, only allow scanning once it's finished?